generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -298,7 +316,7 @@ spring:
             client-id: web_app
             client-secret: web_app
   <%_ } _%>
-            scope: openid,profile,email
+            scope: openid, profile, email, offline_access # last one for refresh tokens
```

Comment:
It's the only way to refresh an expired access token. Also, it looks to be mainstream: https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#offline_access

Reply:
Yes, but since the token is kept in the backend session, not in the client, there is no use for renewing expired tokens.
If we increase the clockSkew to match session timeout, the session token will never expire.

It's useful for ionic.